app/models/concerns/fae/base_model_concern.rb

```diff
@@ -2,12 +2,38 @@ module Fae
   module BaseModelConcern
     extend ActiveSupport::Concern
     require 'csv'
+    require 'slack-notifier'
 
     attr_accessor :filter
 
     included do
       include Fae::Trackable if Fae.track_changes
       include Fae::Sortable
+      after_create :notify_initiation
+      before_save :notify_changes
+    end
+
+    def notify_changes
+      return unless notifiable_attributes.present?
+      notifiable_attributes.each do |field_name_symbol|
+        if self.send("#{field_name_symbol}_changed?") && self.send(field_name_symbol).present?
```

Comment:
Confirming we don't want to notify if a field value was deleted?

Reply:
Not at this time, but def something that could come up in future